release-notes/azure/v11.2.1.md

```diff
@@ -1,13 +1,16 @@
 ## :zap: Giant Swarm Release 11.2.1 for Azure :zap:
 
+### This release contains Kubernetes ```1.16.8``` and other changes that increase cluster resiliency against API Throttling (error 429). Additional improvements include:
+- **Failed provisioning of VMs does not cause 429 anymore.**  We have added an automation that will reimage the failed instance immediately, before it exhausts API rate limits.
+- **Freeze of upgrade if API rate limits are almost exhausted.** Upgrades are causing a movement in deployments, updating templates as well as nodes, triggering more calls to API. Upgraded will be freezed automatically when limits are almost reached and resumed after situation is back to normal.
```

Comment:
```suggestion
- **Freeze of upgrade if API rate limits are almost exhausted.** Upgrades are causing a movement in deployments, updating templates as well as nodes, triggering more calls to the API. Upgraded resources will be frozen automatically when limits are almost reached, and resumed after situation is back to normal.
```

Reply:
I can see I worded it incorrectly, but it is about the whole upgrade process that will be frozen, can I add resources there?

Reply:
I'm not 100% sure about the `Upgraded resources` part, I saw that there was something missing, but I  was hoping you'd know what is actually missing 😬

Reply:
> @axbarsan (didn't know how to do it better so I edited your suggestion, sorry!)
> 
> > Upgrade process will be frozen automatically
> 
> Is that fine?

Yes, that's perfect 👍

Reply:
Perfect, thank you very much for your suggestions!! 😁 

freezed - this will now haunt me 🤦‍♀